# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../qmail-autoresponder/with-mysql.diff
# Copyright (C) 2011 The OpenSDE Project
# Copyright (C) 2006 The T2 SDE Project
# Copyright (C) 1998 - 2004 Clifford Wolf
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./Makefile.orig	2008-04-29 01:34:57.000000000 +0300
+++ ./Makefile	2011-04-15 19:26:12.712726486 +0300
@@ -6,7 +6,7 @@
 
 DEFAULT: all
 
-all: libraries programs docs
+all: libraries programs mysql docs
 
 clean: TARGETS
 	rm -f `cat TARGETS`

